Auto add error logging to all forms

ppamment opened the pull request:
You'll probably want to undo the change to composer.json

Commit:
added entity and changed the repository name to be prefixed with lendable

Commit:
fixed parameter value

Commit:
renamed entity name

Commit:
used a form extension to subscribe the listener instead of injecting on every form individually

Commit:
added a created at value to the entity

Commit:
renamed field createdAt to just created

ollietb commented on composer.json:
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "oh/form-error-log-bundle",
+    "name": "lendable/form-error-log-bundle",
```
@ppamment can you change this back to oh?

titolendable committed:
Support for Symfony3

titolendable committed:
updated service definitions

titolendable committed:
Set dependency on RequestStack

titolendable committed:
Update services.yml

titolendable committed:
Update ErrorLogSubscriber.php

ollietb commented:
@titolendable once you change the name in composer.json back to "oh/form-error-log-bundle" I'll merge this in

Commit:
added event before storing entity

Commit:
changed entity to a mappedsuperclass so we can redifine it as an entity in the project

ollietb commented on composer.json:
```diff
@@ -1,24 +1,27 @@
-        }
-    },
-    "target-dir": "Oh/FormErrorLogBundle"
+  "name": "lendable/form-error-log-bundle",
```
@titolendable can you change this to oh/form-error-log-bundle?

ben-challis committed:
Removed composer.lock, depend on symfony/symfony for now (should be explicit dependencies)

martin-georgiev committed:
Serialize the errored value as we'd like to catch arrays and other unexpected data types

martin-georgiev committed:
Serialize the errored value as we'd like to catch arrays and other unexpected data types

martin-georgiev committed:
Alias namespace import to avoid conflict with current class name

martin-georgiev committed:
PHP resources cannot be serialized so default in this case to empty string.
Extract tha data serialization into trait so it can be potentially reused. This bumps the minimum supported PHP version to 5.4

martin-georgiev committed:
Remove unused trait import

ppamment committed:
Merge pull request #1 from Lendable/bugfix/php-resources-cant-be-serialized

Bugfix for resource serialization

martin-georgiev committed:
Prepare for sf 4

martin-georgiev committed:
Fix travis script path

martin-georgiev committed:
Bump min supported php version to 5.5

martin-georgiev committed:
Change autoloading strategy to PSR-4

martin-georgiev committed:
Merge pull request #2 from Lendable/feature/symfony4-ready

Support Symfony 4
